index.html

```diff
@@ -9399,7 +9504,7 @@ <h2>Translatable States and Properties</h2>
 	</section>
 	<section id="global_states">
 		<h2>Global States and Properties</h2>
-		<p>Some <a>states</a> and <a>properties</a> are applicable to all host language <a>elements</a> regardless of whether a <a>role</a> is applied. The following global states and properties are supported by all roles and by all base markup elements.</p>
+		<p>Some <a>states</a> and <a>properties</a> are applicable to all host language <a>elements</a> regardless of whether a <a>role</a> is applied. The following global states and properties are supported by all roles and by all base markup elements unless otherwise prohibited. Any global states and properties which are prohibited on any role will be listed in the Prohibited States and Properties section.</p>
 		<p class="placeholder">Placeholder for global states and properties</p>
```

Comment:
If a state or property in this list is prohibited from use on some roles, the script should add "(prohibited from use on some roles)"

Reply:
Sounds editorial to me. I'd like to make this happen as agree this adds clarity. @mcking65 Would you be ok pushing this to a future issue if this were the only remaining comment?

Reply:
(did this in the script in the end)